org.srb2.SRB2Kart-galaxy_azerty.yaml

```diff
@@ -0,0 +1,156 @@
+app-id: org.srb2.SRB2Kart-galaxy_azerty
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: srb2kart.sh
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --socket=pulseaudio
+  - --socket=wayland
```

Comment:
does it support wayland? If yes then line 9 should be `fallback-x11` otherwise this should be removed.